api-specificatie/openapi.yaml

```diff
@@ -1317,14 +1317,14 @@ components:
       properties:
         self:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
-        ligtInWoonplaats:
+        woonplaats:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink'
           description: 'De aan het openbareruimte gerelateerde woonplaats (lnik).'
 
     OpenbareRuimte_embedded:
       type: object
       properties:
-        ligtInWoonplaats:
+        woonplaats:
           $ref: '#/components/schemas/WoonplaatsHalBasis'
           description: 'De aan het oenbareruimte gerelateerde woonplaats (object(en)).'
```

Comment:
Een openbare ruimte kan maar in één woonplaats liggen, meervoudsvorm mag dus weg.

Reply:
Ik neem aan dat je hier de description bedoelt ?

Reply:
Ja klopt.